package.json

```diff
@@ -1,15 +1,33 @@
 {
     "name": "tss-react",
-    "version": "4.0.0",
+    "version": "4.1.0-beta.8",
     "description": "makeStyles is dead, long live makeStyles!",
     "repository": {
         "type": "git",
         "url": "git://github.com/garronej/tss-react.git"
     },
     "main": "dist/index.js",
     "types": "dist/index.d.ts",
+    "module": "dist/esm/index.js",
+    "exports": {
+        ".": {
+            "import": "./dist/esm/index.js"
+        },
+        "./mui": {
+            "import": "./dist/esm/mui.js"
+        },
+        "./nextJs": {
+            "import": "./dist/esm/nextJs.js"
+        },
+        "./compat": {
+            "import": "./dist/esm/compat.js"
+        },
+        "./mui_compat": {
+            "import": "./dist/esm/mui_compat.js"
+        }
```

Comment:
I ended up removing "require" as I don't see the point of keeping them. Agree @jiby-aurum ?

Reply:
@garronej unfortunately I think its required, I tried it 4.1.0 and it fails to import `tss-react/mui` complaining that it is not defined in exports, adding the "require" back fixes them.

Reply:
@jiby-aurum Thank you for the feedback, I added it back in.  
It's in 4.1.1

Reply:
thanks again @garronej just tested, works perfectly !